SPARCore/GADArticle.h

```diff
@@ -1,22 +1,31 @@
 #import <Foundation/Foundation.h>
+#import "GADRemoteModel.h"
 
-@interface GADArticle : NSObject
+@interface GADArticle : GADRemoteModel
 
+@property (nonatomic, strong) NSDate *datePublished;
+@property (nonatomic, strong) NSString *brief;
+@property (nonatomic, strong) NSString *headerImage;
 @property (nonatomic, strong) NSString *publicationId;
+@property (nonatomic, strong) NSDate *dateEdited;
+@property (nonatomic, strong) NSString *articleId;
+@property (nonatomic, strong) NSString *title;
+@property (nonatomic, strong) NSString *content;
 @property (nonatomic, strong) NSArray *authors;
 @property (nonatomic, strong) NSURL *url;
-//@property (nonatomic, strong) NSInteger *pictureSize;
 @property (nonatomic, strong) NSString *series;
 @property (nonatomic, strong) NSArray *tags;
-@property (nonatomic, strong) NSString *brief;
-@property (nonatomic, strong) NSDate *datePublished;
-@property (nonatomic, strong) NSString *content;
-//@property (nonatomic, strong) NSSOMETHING *headerImage;
-@property (nonatomic, strong) NSString *title;
 @property (nonatomic, strong) NSString *issue;
-@property (nonatomic, strong) NSString *articleId;
+
++ (void) articlesFromPublication: (NSString *)publicationId
+               completionHandler:(void(^_Nonnull)(NSArray<GADArticle *>
+                                                  *_Nullable articles,
+                                                  NSError *_Nullable error))completion;
+
 
 + (NSArray <GADArticle *> *) loadDummyArticles;
 
++ (NSArray <GADArticle *> *) articlesFromJSON:(NSData *)json;
```

Comment:
If this method isn't public, we can't test it in the testing suite class we created. How do we get around this problem while keeping the method private? I can't remember, but I know there's a way to do it.